scripts/run-tests.sh

```diff
@@ -11,15 +11,4 @@ if [ "${TESTING_PROFILE}" = "integration-tests" ] || [ "${TESTING_PROFILE}" = "r
     tar xvf secrets.tar
 fi
 
-if [ ${#EXTRA_MAVEN_VAR} -eq 0 ]; then
-        # always do non-coverage builds
-    echo "Non-coverage build"
-	mvn --batch-mode clean install -P${TESTING_PROFILE} ${EXTRA_MAVEN_VAR}
-else
-    mvn --batch-mode dependency:get -Dartifact=org.eluder.coveralls:coveralls-maven-plugin:4.2.0
```

Comment:
Do we no longer need this?

Reply:
Not sure, in the build above it seems like the coveralls-maven-plugin no longer seems to need special treatment

Reply:
Also, if you meant whether we need the branching, travis handles this in stages now 
See 
`if: branch IN (master, develop)`